hail/python/hail/context.py

```diff
@@ -198,9 +210,9 @@ def init(sc=None, app_name='Hail', master=None, local='local[*]',
         Minimum file block size in MB.
     branching_factor : :obj:`int`
         Branching factor for tree aggregation.
-    tmp_dir : :obj:`str`
-        Temporary directory for Hail files. Must be a network-visible
-        file path.
+    tmp_dir : :obj:`str`, optional
+        Networked temporary directory.  Must be a network-visible file
```

Comment:
Not blocking but I find this wordier than necessary. "Must be a network-visible directory path. Default to /tmp." is sufficient I think (also: it's not a file path, and we specify directory in the preceding sentence